llm_on_ray/inference/utils.py

```diff
@@ -194,3 +195,15 @@ def module_import_and_init(module_name, clazz, **clazzs_kwargs):
     module = importlib.import_module(module_name)
     class_ = getattr(module, clazz)
     return class_(**clazzs_kwargs)
+
+
+def parse_jinja_file(chat_template: str):
```

Comment:
should be Optional[str] as you are checking None below, right？

Reply:
Our priority order for chat templates is as follows: user-configured template takes precedence over the model's template, which in turn takes precedence over our default template. Therefore, if the user-configured template is not provided (None), the default template must have a value.

self.predictor.tokenizer.chat_template = (
            parse_jinja_file(self.predictor.infer_conf.model_description.chat_template)
            or self.predictor.tokenizer.chat_template
            or parse_jinja_file(self.predictor.infer_conf.model_description.default_chat_template)
        )